### ISSUE TITLE
Movement Needs to be more Precise

### ISSUE THREAD

**Issue by GhostXb** [ALitttleBitDifferent/AmbientPrologueBugs#47]
Sometimes it feels like I'm running on Ice. The turn radius is too far and it makes running into burning grass too easy and really annoying. Also the healing spell blocks your vision too much. If a game needs to be difficult, it shouldn't be because the controls are hard to master and the visuals get in your way.

**Comment by snoopystallion** [comment]
yeah the running is really bad and a huge flaw. your are constantly drifting. making precise movements impossible. i am constantly running into things. enought footage is here alone
https://youtu.be/9-qvEITejbg

**Comment by Zileris** [comment]
Horses don't really turn on a dime, especially when running.(I personally don't mind some drifting. its a bit fun :P ). 
It does need to be tightened up though, since there are so many ground hazards and platforming needed.
If we get a trot speed implemented, then wild drift running maybe while sprinting. I'll be honest, for me it adds a bit of excitement- like a high speed car chase or something.

**Comment by snoopystallion** [comment]
well thats true that horsies have a larger turn radius. every beeing has a larger turn radius when running. BUT you dont drift. not at all. (except you are running on ice). trotting would be good. because the normal walking is just to slow to make any progress in the game. and running is tokio drift and unprecise in tight environments. so yeah. there has to be done something

**Comment by ChristopherS** [comment]
For discussions about how to imiplement trotting, please refer to #5 
This way it's easier to follow the discussion. In general it trotting sounds like a good idea, though we are currently thinking about how to implement it.

As for the issue, I also think the drifting feels unnatural. It doesn't have to be sharp turns, but at the moment it seems that the turn radius is bigger then it would seem from the animation.

**Comment by ChristopherS** [comment]
not sure if it's still a problem. I leave it open for now.
